spec/validators/fee/interim_fee_validator_spec.rb

```diff
@@ -1,7 +1,28 @@
 require 'rails_helper'
 
+# note: before validation hook sets nil to zero
+RSpec.shared_examples 'quantity numericality between' do |min, max|
```

Comment:
**[not a blocker/nit pick]** If you're only using this shared example in this spec, you could just have them inside the top level of `RSpec.describe`:

```ruby
RSpec.describe Fee::InterimFeeValidator, type: :validator do
  shared_examples 'quantity numericality between' do |min, max|
    ...
  end

  ...
end